spec/presenters/contact_presenter_spec.rb

```diff
@@ -23,13 +23,15 @@
     expect(payload[:need_ids]).to be_empty
 
     details = payload[:details]
-    expect(details[:slug]).to eq(contact.slug)
-    expect(details[:title]).to eq(contact.title)
```

Comment:
Apologies for only just noticing this (should've been paying more attention before), but we probably want to leave these in, as we do still want the slug and title of the contact in the details hash so should keep testing that they're present.